releasenotes/notes/sparsepauliop-from-index-list-4660fdaa492cd8b2.yaml

```diff
@@ -0,0 +1,18 @@
+---
+features:
+  - |
+    Support the construction of a :class:`.SparsePauliOp` from a list of Paulis, specifying with
+    an index on which qubits the Pauli act. For example, the operator
```

Comment:
It helps to write these in such a way that we include cross-references to the relevant functions, so the user can click them for more info.  For example, something like:

```rst
The :meth:`.SparsePauliOp.from_list` constructor can now take an iterable where
the elements represent Pauli terms that are themselves sparse, so that
`"XIIIIIIIIIIIIIIIIX"` can now be written as `("XX", [0, 16])`.  For example, the operator
```